pulseapi/entries/views.py

```diff
@@ -126,11 +128,19 @@ class EntryView(RetrieveAPIView):
     """
     A view to retrieve individual entries
     """
-    queryset = Entry.objects.public()
+
+    #queryset = Entry.objects.public()
+
+    def get_queryset(self):
```

Comment:
So this now allows the front end to get all entries regardless of moderation state for Mozilla staff?